src/clip.sol

```diff
@@ -55,13 +55,13 @@ contract Clipper {
     }
 
     // --- Data ---
-    bytes32 public ilk;  // Collateral type of this Clipper
+    bytes32 immutable public ilk;      // Collateral type of this Clipper
 
-    address    public vow;   // Recipient of dai raised in auctions
-    VatLike    public vat;   // Core CDP Engine
-    DogLike    public dog;   // Liquidation module
-    SpotLike   public spot;  // Collateral price module
-    AbacusLike public calc;  // Current price calculator
+    address              public vow;   // Recipient of dai raised in auctions
+    VatLike    immutable public vat;   // Core CDP Engine
+    DogLike              public dog;   // Liquidation module
+    SpotLike   immutable public spot;  // Collateral price module
+    AbacusLike           public calc;  // Current price calculator
```

Comment:
Suggestion: separate `immutable` from mutable values for better organizatoin/clarity.